File: app/src/main/java/com/example/mutidemo/util/callback/ICompressListener.java
package com.example.mutidemo.util.callback;

import java.io.File;

public interface ICompressListener {
    /**
     * Fired when a compression returns successfully, override to handle in your own code
     */
    void onSuccess(File file);

    /**
     * Fired when a compression fails to complete, override to handle in your own code
     */
    void onError(Throwable e);
}

File: app/src/main/java/com/example/mutidemo/callback/ICompressListener.kt
package com.example.mutidemo.callback

import java.io.File

interface ICompressListener {
    /**
     * Fired when a compression returns successfully, override to handle in your own code
     */
    fun onSuccess(file: File)

    /**
     * Fired when a compression fails to complete, override to handle in your own code
     */
    fun onError(e: Throwable)
}